scalafmt-core/shared/src/main/scala/org/scalafmt/config/SortSettings.scala

```diff
@@ -0,0 +1,53 @@
+package org.scalafmt.config
+
+import metaconfig._
+
+@DeriveConfDecoder
+case class SortSettings(
+    order: Vector[SortSettings.ModKey]
+)
+
+object SortSettings {
+
+  implicit val SortSettingsModKeyReader: ConfDecoder[ModKey] =
+    ReaderUtil.oneOfIgnoreBackticks[ModKey](
+      `implicit`,
+      `final`,
+      `sealed`,
+      `abstract`,
+      `override`,
+      `private`,
+      `protected`,
+      `lazy`,
+    )
+
+  val defaultOrder: Vector[ModKey] =
```

Comment:
is there any reason why the default order doesn't follow standard scala style:

https://docs.scala-lang.org/style/declarations.html#modifiers

Specifically, `final` should precede `lazy`.

Reply:
I have no strong preference, defaulting to the scala style guide sounds good 👍